src/ovms.h

```diff
@@ -528,6 +553,28 @@ void OVMS_InferenceResponseDelete(OVMS_InferenceResponse* response);
 // \return OVMS_Status object in case of failure
 OVMS_Status* OVMS_Inference(OVMS_Server* server, OVMS_InferenceRequest* request, OVMS_InferenceResponse** response);
 
+// Execyte asynchronous inference
```

Comment:
```suggestion
// Execute asynchronous inference
```
does the new spelling checker work?